_haikus/cloud_cloud.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: Haiku
```

Comment:
This needs to be lower-case to be interpreted correctly, please amend this line to:

```
layout: haiku
```